packages/components/src/components/Snackbar/Snackbar.stories.tsx

```diff
@@ -34,27 +35,48 @@ export default meta
 
 type Story = StoryObj<SnackbarProps>
 
+const isWeb = Platform.OS !== 'web'
+
 const mobileComponentRenderer = (props: SnackbarProps) => {
-  const { isError, messageA11y, onActionPressed } = props.data || {}
+  const { isError, messageA11y, onActionPressed, offset } = props.data || {}
+  const { show } = useSnackbar()
   const onPressSnackbar = () => {
-    ShowSnackbar(props.message as string, {
+    show(props.message as string, {
       isError,
       messageA11y,
       onActionPressed,
+      offset,
     })
   }
 
-  return <Button label="Press for Snackbar" onPress={onPressSnackbar} />
+  return (
+    <>
+      <Button label="Press for Snackbar" onPress={onPressSnackbar} />
+    </>
+  )
+}
```

Comment:
Was this from our weird inconsistency in auto formatting or was it part of draft testing that can be rolled back? At a minimum the `<>` and `</>` seem unneeded since it's only a single component.